owncloudData/src/main/java/com/owncloud/android/data/server/datasources/implementation/OCRemoteServerInfoDataSource.kt

```diff
@@ -41,9 +41,8 @@ class OCRemoteServerInfoDataSource(
         // Use the same client across the whole login process to keep cookies updated.
         val owncloudClient = clientManager.getClientForAnonymousCredentials(path, false)
 
-        // Step 1: check whether the root folder exists.
-        val checkPathExistenceResult =
-            serverInfoService.checkPathExistence(path, isUserLogged = false, client = owncloudClient)
+        var checkPathExistenceResult =
+            serverInfoService.checkPathExistence(path, isUserLoggedIn = false, owncloudClient)
```

Comment:
Do not remove the comment with step 1 if we keep the step 2 😉

BTW, I would keep the named params